.github/workflows/lint.yml

```diff
@@ -30,19 +30,17 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
-        pip install .[dev,all]
-
+        poetry install --all-extras
     - name: check-sort-import
       run: |
-        make check-sort-imports
-
+        poetry run check-sort-imports
     - name: check-black-format
       run: |
-        make check-format
-
-    - name: check-mypy
-      run: |
-        make mypy
+        poetry run check-format
```

Comment:
@tylerhutcherson I forgot I commented `mypy` out since it was failing both locally and on CI. I'll investigate what the deal is.

Reply:
This is fixed now